scalability_and_performance/index.adoc

```diff
@@ -0,0 +1,63 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index"]
+= {product-title} Scalability and Performance overview
+include::_attributes/common-attributes.adoc[]
+:context: index
+
+// ifndef::openshift-dedicated,openshift-rosa[]
+{product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. This book in the documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.
+
+To get support on these topics, refer to the section of the documentation entitled xref:../support/getting-support.adoc#getting-support[Getting support]
+
+// endif::openshift-dedicated,openshift-rosa[]
+
+[NOTE]
+====
+Operators described in this section have release cycles that are separate from the core {product-title} release cycles. The release schedule for the Node Observability Operator can be found here link:access.redhat.com/support/policy/updates/openshift_operators[Openshift Operators]
+====
+
+The Scalability and Performance section of the documentation offers the following topics:
```

Comment:
I would personally skip this sentence entirely.

Reply:
Done. Glad to!